doc/descriptors.md

```diff
@@ -0,0 +1,95 @@
+# Support for Output Descriptors in Bitcoin Core
+
+Since Bitcoin Core v0.17, there is support for Output Descriptors in the
+`scantxoutset` RPC call. This is a simple language which can be used to
+describe collections of output scripts.
+
+This document describes the language. For the specifics on usage for scanning
+the UTXO set, see the `scantxoutset` RPC help.
+
+## Features
+
+Output descriptors currently support:
+- Pay-to-pubkey scripts (P2PK), through the `pk` function.
+- Pay-to-pubkey-hash scripts (P2PK), through the `pkh` function.
+- Pay-to-witness-pubkey-hash scripts (P2WPK), through the `wpkh` function.
+- Pay-to-script-hash scripts (P2SH), through the `sh` function.
+- Pay-to-witness-script-hash scripts (P2WSH), through the `wsh` function.
+- Multisig scripts, through the `multi` function.
+- Any type of supported address (P2PKH, P2SH, Bech32), through the `addr` function.
```

Comment:
add P2PK as well

Reply:
No, it isn't. There is no address format for P2PK.

Reply:
huh, curious why Core API always returns P2PKH address for them, at least for coinbase outputs.

TIL.

Reply:
@instagibbs That's due to a historical confusion where "address" and "pubkey identifier" were conflated. You can compute the hash160 of pubkeys, and those used to be represented in P2PK address format. For a long time for example `validateaddress` used to do that (it would list the P2PKH address for the pubkeys in a multisig too, for example, which makes no sense at all).

As an address, the P2PKH address format refers specifically to P2PKH scripts.